src/main/java/ratpack/zipkin/internal/DefaultClientTracingInterceptor.java

```diff
@@ -0,0 +1,59 @@
+package ratpack.zipkin.internal;
+
+import brave.Span;
+import brave.http.HttpClientHandler;
+import brave.http.HttpTracing;
+import brave.propagation.TraceContext;
+import com.google.common.reflect.TypeToken;
+import ratpack.exec.Execution;
+import ratpack.func.Action;
+import ratpack.http.MutableHeaders;
+import ratpack.http.client.HttpResponse;
+import ratpack.http.client.ReceivedResponse;
+import ratpack.http.client.RequestSpec;
+import ratpack.zipkin.ClientTracingInterceptor;
+
+import javax.inject.Inject;
+
+public class DefaultClientTracingInterceptor implements ClientTracingInterceptor {
+
+  private static final TypeToken<Span> SpanToken = new TypeToken<Span>() {};
+
+  private final HttpClientHandler<RequestSpec, Integer> handler;
+  private final TraceContext.Injector<MutableHeaders> injector;
+  private final Execution execution;
```

Comment:
According to feedback I received from the Ratpack Slack channel, we should be able to rely on the Ratpack Execution to maintain trace context state for us.  While the tests do pass with this configuration, we should still try it on a real application just to be sure.